packages/react-composites/src/composites/CallWithChatComposite/CallWithChatComposite.tsx

```diff
@@ -177,6 +177,18 @@ const CallWithChatScreen = (props: CallWithChatScreenProps): JSX.Element => {
       setActivePane('none');
     } else {
       setActivePane('chat');
+      // timeout is required to give the window time to render the sendbox so we have something to send focus to.
+      // TODO: Selecting elements in the DOM via attributes is not stable. We should expose an API from ChatComposite to be able to focus on the TextBox.
+      const chatFocusTimeout = setInterval(() => {
+        const textBox = document.querySelector(`[id="sendbox"]`) as HTMLTextAreaElement;
```

Comment:
nit: sendbox over textBox for variable name for contextual clarity of what textbox focus is being set (also comments are query refer to sendbox so its nice to keep the same name for consistency)